src/misc/SvgPolyfill.ts

```diff
@@ -0,0 +1,11 @@
+import { InitializationEvents } from '../events/InitializationEvents';
+import { $$ } from '../utils/Dom';
+require('svg4everybody');
+
+declare function svg4everybody(): void;
+
+export function SvgPolyfill(root: HTMLElement) {
+  $$(root).on(InitializationEvents.afterComponentsInitialization, () => {
```

Comment:
Does this absolutely need to be done `afterComponentsInitialization` ?

For example, we Promise polyfill, we do it when the scripts are loaded.